media-libs/openimageio/patches/openimageio-2.3.10.0.patchset

```diff
@@ -22,21 +22,6 @@ index a1ea8bd..9015ce7 100644
  #else
  
  #undef bswap_32
-diff --git a/src/libOpenImageIO/CMakeLists.txt b/src/libOpenImageIO/CMakeLists.txt
-index 699d592..059af0f 100644
---- a/src/libOpenImageIO/CMakeLists.txt
-+++ b/src/libOpenImageIO/CMakeLists.txt
-@@ -166,7 +166,9 @@ if (MINGW)
-     target_link_libraries (OpenImageIO PRIVATE ws2_32)
- endif()
- 
--
-+if (__HAIKU__)
```

Comment:
Shouldn't it be `if (HAIKU)`

Reply:
Nice catch, checking :)

Reply:
Needed PRIVATE too in there for conflicting target, thanks on the pointer!